### DISCUSSION TITLE
Can any current admin or repository owner provide me answers for these?

### DISCUSSION THREAD

**Discussion by ProphetDaniel** [ethereumclassic/ethereumclassic.github.io#695] [General]
I reached out to Github to understand the case

>I was once the topmost committer to https://github.com/ethereumclassic/ethereumclassic.github.io.
>
>Can you help me understand what has happened?
>Why my contribution to Ethereum Classic website is not present there?
>
>Can you provide me a report with information about who are responsible for this history rewrite?

And I was answered:

>Hi ProphetDaniel,
>
>Thanks for reaching out! Hmm, I'm not seeing any commits made by your account in the >ethereumclassic/ethereumclassic.github.io repository. If you do have commits there but your account is not showing in the >Contributors tab, could you please send over a link to an example commit from your account?
>
>If commit metadata was altered in the repository I'm afraid we here at GitHub Support do not have any insight into that, and >we can only provide log details to users who are set as admins or owners of the repo in question. For a situation like you are >experiencing I recommend reaching out to the maintainer of that repository for more information.
>
>Sorry that we could not be of more assistance here! Please let me know if you have any further questions at all.
>
>Cheers,
>Alex

**Comment by ProphetDaniel** [comment]
We decided to keep on the original chain for this very same reason.
> History is not supposed to be rewritten.

Now we had a situation like this opposed to one of the core principles we fought hard for stablishing.

**Comment by IstoraMandiri** [comment]
Hi @ProphetDaniel 

Your contributions on the old version are not lost - we archived the old version for record:

https://github.com/ethereumclassic/ethereumclassic.github.io-v1

Here's an example of your commits https://github.com/ethereumclassic/ethereumclassic.github.io-v1/commits/master-old?after=94909698580300fe36da2ab241aa19b5ed8c4c5d+104&branch=master-old

**Comment by ProphetDaniel** [comment]
Dear @hitchcott, I remember @dax-classic overwrote many of my commits when the first Gatsby version was launched can you ask for GitHub the logs of that rewrite operation?

**Comment by IstoraMandiri** [comment]
@ProphetDaniel 

I can see commits from you in that repo from 27th July 2016, which was only a week after the ETC hard fork.

Are older commits missing?

https://github.com/ethereumclassic/ethereumclassic.github.io-v1/commits/gh-pages

From what I can tell there wasn't a "rewrite operation", it's on a different branch.

**Comment by ProphetDaniel** [comment]
> @ProphetDaniel
> 
> I can see commits from you in that repo from 27th July 2016, which was only a week after the ETC hard fork.
> 
> Are older commits missing?
> 
> https://github.com/ethereumclassic/ethereumclassic.github.io-v1/commits/gh-pages
> 
> From what I can tell there wasn't a "rewrite operation", it's on a different branch.

GitHub is able to provide evidences in this regard.

One repository owner has to ask for the information, though.
I had many commits to the repository then there was a merge which overwrote history by @dax-classic.
Can you provide me with the logs GitHub provides?

**Comment by IstoraMandiri** [comment]
Hi @ProphetDaniel can you provide more details about which merge this was?

As mentioned, as far as I can tell all the commits made to both versions were preserved in various branches, and none have disappeared. 

If you can provide a specific commit that you know for sure is missing then that would help figure out what happened.

**Comment by ProphetDaniel** [comment]
> Hi @ProphetDaniel can you provide more details about which merge this was?

What is stopping you form asking for GitHub logs? For a reason my contributions to the repository were deleted. See this: https://github.com/ethereumclassic/ethereumclassic.github.io-v1/graphs/contributors

**Comment by IstoraMandiri** [comment]
Okay, I think I understand what's going on now. 

It's not that your commits aren't showing up or that your contributions were deleted; it's just that your account isn't appearing on the ranking on contributions page.

You are appearing in the _contributions sidebar_ in that repo, just not on the main page, which makes me think...

![image](https://user-images.githubusercontent.com/1696942/118900029-40b86480-b93a-11eb-93f7-a59d97a09387.png)

I checked the old commits, and they were committed by 

**Author: Prophet Daniel <prophetdaniel@ethereumclassic.org>**

Notice [here](https://github.com/ethereumclassic/ethereumclassic.github.io-v1/commit/33012ea4bf66da778d2fefb7a3948555a9b28c6f?branch=33012ea4bf66da778d2fefb7a3948555a9b28c6f&diff=unified) that your account name isn't linkable. Compare that to one of your [recent commits](https://github.com/ProphetDaniel/DeclarationOfIndependence/commit/b7b6663aa4b31c6e4bb9d6d35b33f219c29116f5)

**Committer: Prophet Daniel <daniel_the_prophet@protonmail.com>**

Are you using the same email for your github account? This is probably be what is causing the link to break, and therefor not be listed on the contributors page (which only shows known github users I believe)?

Related issue: https://stackoverflow.com/questions/15289768/github-commits-arent-recorded-in-the-your-contributions-calendar

I don't want to reach out to github support immediately because without diagnosing the problem I don't see how getting those logs are going to help, and it will be wasting their time unless we have a good reason to access them.

Btw, their response about not seeing your commits is related to the new repo, which indeed does not have any of your commits as you did not commit to that repo -- so perhaps you can ask them the same question about the v1 repo, explain the email thing. I'm fairly sure it'll have something to do with the email address you used.

**Comment by IstoraMandiri** [comment]
Btw it seems you can add multiple emails to your github account, have you tried this? https://github.blog/2008-02-28-multiple-emails/

**Comment by IstoraMandiri** [comment]
I also [found](https://duckduckgo.com/?q=github+contributions+not+appearing) this https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/managing-contribution-graphs-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile

Please can you check that and make sure it's all as expected?

**Comment by ProphetDaniel** [comment]
I don't have this email anylonger: `prophetdaniel at ethereumclassic.org`

> I don't want to reach out to github support immediately because without diagnosing the problem I don't see how getting those logs are going to help, and it will be wasting their time unless we have a good reason to access them.

Good reason to access them is history rewrite suspicion.

> Btw it seems you can add multiple emails to your github account, have you tried this? https://github.blog/2008-02-28-multiple-emails/

I just added the email I had `prophetdaniel at ethereumclassic.org`. But I cannot access that mailbox anymore in order to verify it.

> Btw, their response about not seeing your commits is related to the new repo, which indeed does not have any of your commits as you did not commit to that repo -- so perhaps you can ask them the same question about the v1 repo, explain the email thing. I'm fairly sure it'll have something to do with the email address you used.

Let me check this with GitHub....

**Comment by ProphetDaniel** [comment]
After adding as a secondary email it now looks alright...

![Imgur](https://i.imgur.com/AZN5YV6.png)

**Comment by ProphetDaniel** [comment]
Now the only missing piece of the Puzzle is the disappeared commits before Dax Classic overwrite.

**Comment by IstoraMandiri** [comment]
I have moved this to a discussion to keep the issues organized for development taks
